packages/mock-block-dock/package.json

```diff
@@ -43,8 +43,8 @@
     "@emotion/styled": "^11.10.0",
     "@lit-labs/react": "^1.0.4",
     "@mui/material": "^5.10.4",
-    "ajv": "^8.9.0",
-    "echarts": "^5.3.3",
+    "ajv": "patch:ajv@npm%3A8.9.0#~/.yarn/patches/ajv-npm-8.9.0-d7480193a9.patch",
+    "echarts": "patch:echarts@npm%3A5.3.3#~/.yarn/patches/echarts-npm-5.3.3-e3b4e49969.patch",
```

Comment:
Needs resolving. `mock-block-dock` is being published, so if it depends on a patched package, it won’t work when installed via npm.